eng/tools/lint-diff/src/lint-diff.ts

```diff
@@ -102,11 +103,25 @@ async function runLintDiff(
   baseBranch: string,
   compareSha: string,
 ) {
-  const [beforeList, afterList, affectedSwaggers] = await getRunList(
-    beforePath,
-    afterPath,
-    changedFilesPath,
-  );
+  let beforeList, afterList, affectedSwaggers;
+  try {
+    [beforeList, afterList, affectedSwaggers] = await getRunList(
+      beforePath,
+      afterPath,
+      changedFilesPath,
+    );
+  } catch (error) {
+    process.exitCode = 1;
+    console.log(error);
+
```

Comment:
```suggestion
    process.exitCode = 1;
    console.log(error);

```

replaced by re-throw